src/TextForEvent.tsx

```diff
@@ -729,6 +746,26 @@ function textForMjolnirEvent(event: MatrixEvent): () => string | null {
         "for %(reason)s", { senderName, oldGlob: prevEntity, newGlob: entity, reason });
 }
 
+export function textForLocationEvent(event: MatrixEvent): () => string | null {
+    return () => _t("%(senderName)s has shared their location", {
+        senderName: getSenderName(event),
+    });
+}
+
+function textForPollStartEvent(event: MatrixEvent): () => string | null {
+    return () => _t("%(senderName)s has started a poll - %(pollQuestions)s", {
+        senderName: getSenderName(event),
+        pollQuestions: event.getContent()[M_POLL_START.name].question[TEXT_NODE_TYPE.name],
```

Comment:
also shouldn't both of these be singular? It's meant to be just one `question`, not multiple.